cPP_DSC.adoc

```diff
@@ -294,7 +294,7 @@ DSCs are implemented to satisfy the following use cases:
 
 *[USE CASE 1] Protected Key Store*
 
-A platform leveraging DSCs as a hardware-secured Private Key Store facilitates the use of secure and protected storage of secret symmetric keys and private asymmetric keys for access to data and services. These DSCs would provide safe use of the private and secret keys inside the protected hardware boundary.
+A platform leveraging DSCs as a hardware-secured Key Store facilitates the use of secure and protected storage of symmetric and asymmetric keys for access to data and services. These DSCs would ensure the confidentiality and integrity of the secret and private keys, as well as the integrity of public keys, inside the protected hardware boundary.
```

Comment:
In my opinion, a protected key store can also include public keys, with the purpose of protecting the integrity of these keys.